security/keycloak/README.md

```diff
@@ -51,38 +50,79 @@ docker run -it -d \
   jboss/keycloak:3.4.0.Final \
   -b 0.0.0.0 \
   -Dkeycloak.migration.action=import \
-  -Dkeycloak.migration.provider=singleFile \
-  -Dkeycloak.migration.file=/tmp/realm/wildfly-swarm-keycloak-example-realm.json
+  -Dkeycloak.migration.provider=dir \
+  -Dkeycloak.migration.file=/tmp/realm
 ```
 
-## Build & Run Example
+## Build Example
 
 ``` sh
 mvn clean package
-java -jar target/example-keycloak-swarm.jar
 ```
 
-## Access a secured resource
+## Access a secured resource from shell
+
+### Start the example server
+
+``` sh
+THIS_EXAMPLE=/path/to/this/example
+java -Dswarm.keycloak.json.path=$THIS_EXAMPLE/src/main/resources/cmd-client-keycloak.json \
+  -jar target/example-keycloak-swarm.jar
+```
+
+### Access the secured resource
 
 ``` sh
-curl localhost:8080/secured -v
+curl localhost:8080/app/secured -v
 ```
 
-You'll get the response with `401 Unauthorized` . Let's get a Token to access it.
+You'll get the response with `401 Unauthorized`. Let's get a Token to access it.
 
-## Obtain Token from Keycloak Server
+### Obtain Token from Keycloak Server
 
 ``` sh
 USER=user1
 PASS=password1
-RESULT=`curl -s --data "grant_type=password&client_id=curl&username=${USER}&password=${PASS}" http://localhost:8180/auth/realms/wildfly-swarm-keycloak-example/protocol/openid-connect/token`
+RESULT=`curl -s --data "grant_type=password&client_id=wildfly-swarm-cmd-client-example&username=${USER}&password=${PASS}" http://localhost:8180/auth/realms/wildfly-swarm-cmd-client/protocol/openid-connect/token`
 TOKEN=`echo $RESULT | sed 's/.*access_token":"//g' | sed 's/".*//g'`
 ```
 
-## Access the secured resource with the Token
+### Access the secured resource with the Token
 
 ``` sh
-curl -H "Authorization: bearer $TOKEN" localhost:8080/secured
+curl -H "Authorization: bearer $TOKEN" localhost:8080/app/secured
 ```
 
 You'll get the response which contains `Hi user1, this is Secured Resource`.
+
+## Access a secured resource from browser
+
+### Start the example server
+
+Stop the example server if it is already running and start it with:
+
+``` sh
+THIS_EXAMPLE=/path/to/this/example
+java -Dswarm.keycloak.json.path=$THIS_EXAMPLE/src/main/resources/browser-client-keycloak.json \
+  -jar target/example-keycloak-swarm.jar
+```
+
+### Access the public client page
+
+Navigate the browser to "http:localhost:8080/" 
+
+### Access the secured resource
+
+Click on "Access Secured Resource"
+You'll get `Unauthorized` displayed.
+
+### Login
+
+Click on "Login" and use the "user1:password1" credentials to login into Keycloak.
```

Comment:
I'm not sure

> "user1:password1"

is a common format. Might be slightly misleading. I'd use

> `user1` / `password1`

Your call :-)